### PR TITLE
trigger configurability, always pushback

### PR DESCRIPTION
**Pull request by ddobrigk**

### CODE REVIEW

File: HLT/VZERO/CMakeLists.txt [hunk 1]
```diff
@@ -22,6 +22,7 @@ include_directories(${AliRoot_SOURCE_DIR}/HLT/VZERO)
 # Additional include folders in alphabetical order except ROOT
 include_directories(SYSTEM ${ROOT_INCLUDE_DIR})
 include_directories(${AliRoot_SOURCE_DIR}/HLT/BASE
+		    ${AliRoot_SOURCE_DIR}/HLT/BASE/util
```
Comment (dberzano) [inline HLT/VZERO/CMakeLists.txt]: Please use spaces for indent, not tabs

**Comment by ddobrigk** [comment]
Hi @dberzano , I think I did the modification you requested here (f29579a, see above) but I am not sure if this is enough (never used this on-the-fly web-browser-based code editing before...)... Did I do it correctly? Is there something else missing from my side? Thanks!

**Comment by dberzano** [comment]
You did right, but I am not the right person to approve this PR. Maybe @chiarazampolli is?

**Comment by chiarazampolli** [comment]
+test

**Comment by chiarazampolli** [comment]
Hello,

I can approve for merging, but this is more HLT-oriented, so maybe @mkrzewic  wants to approve it? Otherwise I can do it - it seems fine with me.

Chiara

**Comment by davidrohr** [comment]
+1 seems ok from hlt side, i will deploy it on the farm tomorrow if possible
Kind Regards
David Rohr

(Sent from my mobile, excuse the typos.)

**Comment by mkrzewic** [comment]
+1

**Comment by mkrzewic** [comment]
actually this will not parse the options, since a call to AliOptionParser::ProcessOptionString is missing

**Comment by ddobrigk** [comment]
Hi @mkrzewic , thanks for pointing that out! Now added...

**Comment by ddobrigk** [comment]
@davidrohr , really not sure how I managed to do this, sorry... But I think the cleanest (and easiest for me) is if we close this pull request (without accepting) and I open another one in which I really only include the changes to the online vzero module and nothing else. I already prepared it by rebasing my AliRoot to master and will close this PR and the new one will be straightforward, I hope.

**Pull request closed without merging**